plugins/catalog-import/src/api/CatalogImportClient.ts

```diff
@@ -114,6 +118,24 @@ export class CatalogImportClient implements CatalogImportApi {
     };
   }
 
+  preparePullRequest(): {
```

Comment:
does this not want to be async? for the future

Reply:
😆 Yeah, thought that too. The return values are used as the initial state. The current usage requires it to be sync, but I think I could rewrite that a bit.

Reply:
Here we go, now it's sync